### ISSUE TITLE
qt: Build error

### ISSUE THREAD

**Issue by sidaf** [Homebrew/linuxbrew#427]
Hi,

I'm trying to build the QT formula, but compilation [fails](https://gist.github.com/9e191045c54339748a9a) on my Fedora 22 based system. It seems to be a known issues with [Fedora/Redhat](https://bugreports.qt.io/browse/QTBUG-5385), but no fixes have been committed. Arch Linux have included a fix by modifying the [LD_LIBRARY_PATH variable](https://projects.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/qt4#n76) as suggested in the bug report. Can a similar change be made to the QT formula?

Regards,

Sion

**Comment by sjackman** [comment]
https://gist.github.com/9e191045c54339748a9a
```
g++ -L/home/sion/.linuxbrew/lib -Wl,-rpath,/home/sion/.linuxbrew/lib -Wl,-O1 -Wl,-rpath,/home/sion/.linuxbrew/Cellar/qt/4.8.6/lib -Wl,-rpath,/home/sion/.linuxbrew/Cellar/qt/4.8.6/lib -o ../../../../bin/assistant .obj/release-shared/fontpanel.o .obj/release-shared/aboutdialog.o .obj/release-shared/bookmarkdialog.o .obj/release-shared/bookmarkfiltermodel.o .obj/release-shared/bookmarkitem.o .obj/release-shared/bookmarkmanager.o .obj/release-shared/bookmarkmanagerwidget.o .obj/release-shared/bookmarkmodel.o .obj/release-shared/centralwidget.o .obj/release-shared/cmdlineparser.o .obj/release-shared/contentwindow.o .obj/release-shared/findwidget.o .obj/release-shared/filternamedialog.o .obj/release-shared/helpenginewrapper.o .obj/release-shared/helpviewer.o .obj/release-shared/indexwindow.o .obj/release-shared/installdialog.o .obj/release-shared/main.o .obj/release-shared/mainwindow.o .obj/release-shared/preferencesdialog.o .obj/release-shared/qtdocinstaller.o .obj/release-shared/remotecontrol.o .obj/release-shared/searchwidget.o .obj/release-shared/topicchooser.o .obj/release-shared/xbelsupport.o .obj/release-shared/collectionconfiguration.o .obj/release-shared/openpagesmodel.o .obj/release-shared/globalactions.o .obj/release-shared/openpageswidget.o .obj/release-shared/openpagesmanager.o .obj/release-shared/openpagesswitcher.o .obj/release-shared/helpviewer_qtb.o .obj/release-shared/moc_fontpanel.o .obj/release-shared/moc_aboutdialog.o .obj/release-shared/moc_bookmarkdialog.o .obj/release-shared/moc_bookmarkfiltermodel.o .obj/release-shared/moc_bookmarkmanager.o .obj/release-shared/moc_bookmarkmanagerwidget.o .obj/release-shared/moc_bookmarkmodel.o .obj/release-shared/moc_centralwidget.o .obj/release-shared/moc_contentwindow.o .obj/release-shared/moc_findwidget.o .obj/release-shared/moc_filternamedialog.o .obj/release-shared/moc_helpenginewrapper.o .obj/release-shared/moc_helpviewer.o .obj/release-shared/moc_helpviewer_p.o .obj/release-shared/moc_indexwindow.o .obj/release-shared/moc_installdialog.o .obj/release-shared/moc_mainwindow.o .obj/release-shared/moc_preferencesdialog.o .obj/release-shared/moc_qtdocinstaller.o .obj/release-shared/moc_remotecontrol.o .obj/release-shared/moc_searchwidget.o .obj/release-shared/moc_topicchooser.o .obj/release-shared/moc_openpagesmodel.o .obj/release-shared/moc_globalactions.o .obj/release-shared/moc_openpageswidget.o .obj/release-shared/moc_openpagesmanager.o .obj/release-shared/moc_openpagesswitcher.o .obj/release-shared/qrc_assistant.o .obj/release-shared/qrc_assistant_images.o    -L/tmp/qt20150606-31378-1kzgbcg/qt-everywhere-opensource-src-4.8.6/lib -L/tmp/qt20150606-31378-1kzgbcg/qt-everywhere-opensource-src-4.8.6/plugins/sqldrivers -lQtHelp -L/tmp/qt20150606-31378-1kzgbcg/qt-everywhere-opensource-src-4.8.6/lib -L/usr/X11R6/lib -lQtSql -lQtGui -lQtNetwork -lQtCore -lpthread 
/bin/ld: warning: libQtCLucene.so.4, needed by /tmp/qt20150606-31378-1kzgbcg/qt-everywhere-opensource-src-4.8.6/lib/libQtHelp.so, not found (try using -rpath or -rpath-link)
/tmp/qt20150606-31378-1kzgbcg/qt-everywhere-opensource-src-4.8.6/lib/libQtHelp.so: undefined reference to `QCLuceneIndexWriter::setMergeFactor(int)'
```

**Comment by sjackman** [comment]
I'm afraid I don't have the time to troubleshoot individual formula. If you're able to find a fix, please do submit a pull request. Looking at the `LD_LIBRARY_PATH` fix, you could try `brew edit qt` and add the following line to the top of the `install` function.
```ruby
ENV["LD_LIBRARY_PATH"] = prefix/"lib"
```

**Comment by sidaf** [comment]
Fair enough :-)

Thanks or the pointer, have created a fix, please see the following pull request:

https://github.com/Homebrew/linuxbrew/pull/429

**Comment by sjackman** [comment]
Fixed by #429.
